dbms/src/Storages/KVStore/MultiRaft/IngestSST.cpp

```diff
@@ -180,6 +180,8 @@ void Region::finishIngestSSTByDTFile(RegionPtr && temp_region, UInt64 index, UIn
     {
         std::unique_lock<std::shared_mutex> lock(mutex);
 
+        auto uncommitted_ingest = temp_region->dataSize();
+        GET_METRIC(tiflash_raft_write_flow_bytes, type_ingest_uncommitted).Observe(uncommitted_ingest);
```

Comment:
what about the uncommitted bytes after `applySnapshot`

Reply:
We can find some clue in `tiflash_raft_ongoing_snapshot_total_bytes` here if one snapshot has too many uncommitted data, by `type_raft_snapshot - type_dt_total`.
Of course, this metric is not as good as `tiflash_raft_write_flow_bytes` here. However, `tiflash_raft_write_flow_bytes` is a histogram metric which results in many buckets, I am not really sure if it is worthy here?

Reply:
I'm afraid we can not use `type_raft_snapshot - type_dt_total` to get the remain bytes after applying snapshot. Because `type_dt_total` is the decoded "column" bytes in memory, while `type_raft_snapshot` is the bytes of key-values read from SSTReader.

I think this metrics is to find out the reason why "CurrentMetrics::MemoryTrackingKVStore" grows. There are three known ways and missing it for the apply snapshot is weird:
* normal write raft log
* uncommitted bytes after apply snapshot
* uncommitted bytes after ingest sst

Reply:
Yes, you are right, I will add another metric